Gaming (#518)
solana-developers/solana-cookbook: docs/gaming/hello-world.md

---
title: Hello World example
head:
  - - meta
    - name: title
      content: Solana Cookbook | Hello World example
  - - meta
    - name: og:title
      content: Solana Cookbook | Hello World example
  - - meta
    - name: description

>       content: A tiny adventure on chain game written in anchor with clients in js and c#

Reviewer: ```suggestion
      content: Get started building Solana games with a basic adventure game 
```

  - - meta
    - name: og:description

>       content: A tiny adventure on chain game written in anchor with clients in js and c#

Reviewer: ```suggestion
      content: Get started building Solana games with a basic adventure game
```

  - - meta
    - name: og:image
      content: https://solanacookbook.com/cookbook-sharing-card.png
  - - meta
    - name: og:image:alt
      content: Solana splash card
  - - meta
    - name: twitter:card
      content: summary
  - - meta
    - name: twitter:site
      content: "@solanacookbook"
  - - meta
    - name: twitter:image
      content: "https://solanacookbook.com/cookbook-sharing-card.png"
  - - meta
    - name: robots
      content: index,follow,noodp
  - - meta
    - name: googlebot
      content: index,follow
footer: MIT Licensed
---

> # On chain game

Reviewer: ```suggestion
# Building a game on Solana
```

> ## Write your first Solana game

Reviewer: ```suggestion
## Getting started with your first Solana game
```

Video Walkthrough:
<div class="video-block">
<iframe width="320" height="200" src="https://www.youtube.com/embed/_vQ3bSs3svs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
</div>

Live Version. (use devnet in the embedded version)
<iframe height='400' scrolling='no' title='OZXQWp' src='https://solplay.de/TinyAdventure/index.html' frameborder='no' allowtransparency='true' allowfullscreen='true' style='width: 80%;'>
</iframe>

> ## Tiny Adventure Anchor Program

Reviewer: ```suggestion
## Tiny Adventure
```

> Tiny Adventure is a beginner-friendly Solana program created using the Anchor framework. The goal of this program is to show you how to create a simple game that allows players to set up an account to track their position and move their position left and right.

Reviewer: ```suggestion
Tiny Adventure is a beginner-friendly Solana program created using the Anchor framework. The goal of this program is to show you how to create a simple game that allows players to track their position and move left or right.
```
